spec/features/home_page_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+describe "Home page" do
+	before do
+		visit '/'
+	end
+
+	it "has a link redirecting to About page" do
+		click_link 'About RubyGameDev.com'
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.